.gitignore

```diff
@@ -78,7 +76,4 @@ node/
 node_modules/
 
 # Generated JavaScript Bundles
-jsbundles
-
-# In case someone accidentally runs npm install instead of yarn install
-package-lock.json
```

Comment:
Using NPM is not supported, and having code in the source tree to handle an unsupported configuration by silently ignoring it (rather than failing fast) violates the principle of least surprise.